src/js/components/Layer/LayerContainer.js

```diff
@@ -77,6 +86,28 @@ const LayerContainer = forwardRef(
     }, [position, ref]);
 
     useEffect(() => {
+      const onClickDocument = event => {
+        // determine which portal id the target is in, if any
+        let clickedPortalId = null;
+        let node = event.target;
+        while (clickedPortalId === null && node !== document) {
+          // check if user click occurred within the layer
+          const attr = node.getAttribute('data-l-portal-id');
+          if (attr !== null) clickedPortalId = parseInt(attr, 10);
+          node = node.parentNode;
```

Comment:
add comment that we are looping up to the parent to find if it has the `data-l` attr